manifests/crc/default-db-included-installation/minikube/kruize-crc-minikube.yaml

```diff
@@ -221,6 +221,8 @@ data:
       "isKafkaEnabled" : "false",
       "isROSEnabled": "false",
       "local": "true",
+      "metadataProfileFilePath" : "/home/autotune/app/manifests/autotune/metadata-profiles/bulk_cluster_metadata_local_monitoring.json", 
```

Comment:
@msvinaykumar These were removed earlier from manifest yamls and added into the local test based on Dino's review comments, kruize_local_ros_patch. I think you have removed invocation of this in this PR.